### ISSUE TITLE
Internal error: '[string "PSUploadTask.lua"]:352: attempt to index local 'thumbSettings' (a boolean value)'

### ISSUE THREAD

**Issue by Thoscellen** [flingo64/PhotoStation-Upload-Lr-Plugin#63]
<img width="416" alt="Capture d’écran 2022-07-31 à 14 39 22" src="https://user-images.githubusercontent.com/105607989/182026902-fb12fcf0-0358-45d6-b6a6-1e588bd61bb3.png">

Hello, hope your vacantions went well!

I have the same issue as mentioned on ticket #60 except it's on line 352. I'm trying to upload pictures to Synology Photo without thumbnails.

Can you have a look?

Best Regards.

**Comment by Thoscellen** [comment]
I made a temporary fix based on your suggestion on #62

line 352
```lua
			videoSettings.Orig_Filename, title_Filename, thumbSettings and thumbSettings.XL_Filename, thumbSettings and thumbSettings.L_Filename, thumbSettings and thumbSettings.B_Filename, thumbSettings and thumbSettings.M_Filename, thumbSettings and thumbSettings.S_Filename,
```

**Comment by flingo64** [comment]
Fixed in v7.2.3. Please re-test.
